gatsby-browser.js

```diff
@@ -1,3 +1,10 @@
+const { reportUrlChange, RESET_URL } = require('components/Layout')
+
 exports.onRouteUpdate = ({ location }) => {
   window.analytics && window.analytics.page()
 }
+
+exports.onInitialClientRender = () => {
+  // when leaving website, reset url
+  window.onbeforeunload = () => reportUrlChange(RESET_URL)
```

Comment:
@kravets-levko added this feature so that the "open new window" button would hide when redash.io is left.

This is necessary for a few scenarios. For instance, when navigating away to a csp blocked page - the "open new window" button still shows and holds the previous page:
<img width="411" alt="Screen Shot 2019-06-09 at 13 06 43" src="https://user-images.githubusercontent.com/486954/59157695-c5947580-8ab7-11e9-9694-0207d6afef64.png">

Reply:
I'm not very familiar to Gatsby, so I'll just trust you that this works (and technically it should work) :+1:

Reply:
I tested it locally of course 🤖